### ISSUE TITLE
Abandoned?

### ISSUE THREAD

**Issue by gedw99** [adhocteam/pushup#114]
Seems this repo is dead ?

**Comment by paulsmith** [comment]
No, Pushup is not abandoned nor dead. Just regular life circumstances that have taken my attention away from it for a while. I'm looking forward to getting back into it, especially since it seems htmx has gathered some steam and Pushup goes hand in hand with it, making a compelling frontend server solution.

**Comment by gedw99** [comment]
@paulsmith yep htmx seems to be taking off.

I hope I can help on your project, as I like its design.. Its a different take.

**Comment by riclib** [comment]
yeah, also find it quite elegant. Would love to pitch in too!

**Comment by paulsmith** [comment]
Let's go 🚀

**Comment by gedw99** [comment]
Looks like Nix is being integrated .

glad to see development restart 

could you make a high level roadmap so o can see where I can contribute @paulsmith  ?? 

Maybe a todo.md
